packages/calcite-components/src/components/table/table.tsx

```diff
@@ -496,15 +487,37 @@ export class Table extends LitElement implements LoadableComponent {
         >
           <table
             ariaColCount={this.colCount}
-            ariaMultiSelectable={this.selectionMode === "multiple"}
```

Comment:
Was this throwing an error when role="table"? We can maybe add it back conditionally with if `this.interactionMode === "interactive"` which would be `role="grid"`

Reply:
Good catch. Will fix!

Reply:
Fixed!